core/src/main/scala/org/bitcoins/core/api/dlcoracle/DLCOracleApi.scala

```diff
@@ -62,4 +62,8 @@ trait DLCOracleApi {
   def signDigits(eventName: String, num: Long): Future[OracleEvent]
 
   def signDigits(oracleEventTLV: OracleEventTLV, num: Long): Future[OracleEvent]
+
+  def deleteSigs(eventName: String): Future[OracleEvent]
```

Comment:
Let's add scaladocs at the interface level saying these are potentially dangerous operations that can cause private key leakage.

Reply:
nit: `deleteAttestations()`